scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -447,6 +453,70 @@ def create_branch(
             target_branch, current_release_version_number))
 
 
+def is_invalid_email_present(release_summary_lines):
+    """Checks if any invalid email is present in the list of
+    new authors and contributors.
+
+    Args:
+        release_summary_lines: list(str). List of lines in
+            ../release_summary.md.
```

Comment:
Do we need to be specific here, the variable can be any lines so maybe you can modify it to `List of lines to check.` and maybe add the filename to brackets.

Reply:
Sorry, I did not understand this one, what do you mean by any lines?

Reply:
The `release_summary_lines` can theoretically be lines from different files than ../release_summary.md so I don't see the need to mention that file specifically.

Reply:
No, they will be lines only from the release_summar file i.e. the file being read.

Reply:
What I meant is that in the future this function can be used for different lines from different files and then the docstring would not make sense, so my thinking is to wite the docstring as generic as possible from the starts. But this is not a strong opinion and if you think it is better this way I'm okay with that.

Reply:
Ok, I am keeping it as it is in that case (I think the function should be only used for checking release summary lines given the name of the arg and the script it is used in)